packages/core/src/language-server/glint-language-server.ts

```diff
@@ -416,6 +416,45 @@ export default class GlintLanguageServer {
     return this.glintConfig.environment.isTypedScript(file) ? 'typescript' : 'javascript';
   }
 
+  public organizeImports(
+    uri: string,
+    formatOptions: ts.FormatCodeSettings = {},
+    preferences: ts.UserPreferences = {}
+  ): TextEdit[] {
+    const transformInfo = this.transformManager.findTransformInfoForOriginalFile(
+      uriToFilePath(uri)
+    );
+
+    if (!transformInfo) {
+      return [];
+    }
+
+    const fileTextChanges = this.service.organizeImports(
+      {
+        type: 'file',
+        fileName: transformInfo.transformedFileName,
+        skipDestructiveCodeActions: true,
+      },
+      formatOptions,
+      preferences
+    );
+    const edits: TextEdit[] = [];
+
+    for (const fileTextChange of fileTextChanges) {
+      for (const textChange of fileTextChange.textChanges) {
+        const location = this.textSpanToLocation(fileTextChange.fileName, textChange.span);
+        if (location) {
+          edits.push({
+            range: location.range,
```

Comment:
It's unusual for imports to appear anywhere other than the top of a file, but if they do, I believe TS will sort each group in-place.

If imports appear after a transformed span, then I think this range will be off—if so, it likely needs to be converted using something like `getOriginalRange` to convert from the 'transformed' version TS sees.

Reply:
`this.textSpanToLocation` calls `getOriginalRange` internally to account for this. 

Manually confirmed that `import`s after `<template>` tags in `.gts` files sort correctly and added this scenario to the test cases